routes.js

```diff
@@ -0,0 +1,20 @@
+async function routes (fastify, options) {
+
+    // health checkpoint
+    fastify.get('/health', async (request, reply) => {
+        reply.code(200);
+        return {status: "ok"}
```

Comment:
you can chain the commands like that if you prefer:
```suggestion
        return reply.code(200).send({ status: "ok" })
```